dev-lang/erlang/erlang-21.0.4.ebuild

```diff
@@ -21,25 +21,25 @@ LICENSE="Apache-2.0"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~x64-solaris"
 
-IUSE="dirty-schedulers doc emacs hipe java kpoll libressl odbc smp sctp ssl systemd tk wxwidgets"
-REQUIRED_USE="dirty-schedulers? ( smp )" #621610
+IUSE="doc emacs +hipe java +kpoll libressl odbc +pgo sctp ssl systemd tk +wxwidgets"
 
 RDEPEND="
+	emacs? ( virtual/emacs )
+	java? ( >=virtual/jdk-1.2:* )
```

Comment:
Please remove the explicit version here. `1.2` is gone for years now, and I seriously doubt you've actually tested it.

Reply:
done